boa_tester/src/results.rs

```diff
@@ -50,12 +50,45 @@ impl From<ResultInfo> for ReducedResultInfo {
     }
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize)]
+struct FeaturesInfo {
+    #[serde(rename = "c")]
+    commit: Box<str>,
+    #[serde(rename = "u")]
+    test262_commit: Box<str>,
+    #[serde(rename = "n")]
+    suite_name: Box<str>,
+    #[serde(rename = "f")]
+    features: Vec<String>,
+}
+
+fn remove_duplicates(features_vec: &Vec<String>) -> Vec<String> {
```

Comment:
This should solve the clippy error, but might need some changes in the code to convert the slice to a vector.

```suggestion
fn remove_duplicates(features_vec: &[String]) -> Vec<String> {
```

Reply:
Done. https://github.com/boa-dev/boa/pull/1980/files#diff-fc3d7ad7b5e64574258c9febbe56171f3309b74e0c8da35238a76002f3ee34d9R77